db/migrate/20230118133259_add_users_to_comment_and_add_posts_to_comment.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsersToCommentAndAddPostsToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [ ] Your foreign is `users_id` in the schema.rb file, instead of `author_id` You can point Rails to the users table using the `to_table` attribute. Also, note that it should be singular `author`, not `authors`.

  You can refer to [this Stackoverflow answer](https://stackoverflow.com/a/42056089) for idea on how to reference column in a different table